core/scss/ng/page-header-admin.scss

```diff
@@ -35,5 +35,11 @@
         margin-right: $page-header-icon-margin-right;
       }
     }
+    .control-hub-refresh {
```

Comment:
[question] don't know any context here, but seems this is only specific to CH, why not just add the style in CH? Cause from my understanding, I think `momentum-ui` should be more general instead of correlated with some products. thanks

Reply:
Not disagreeing at all but it turns out the entire file is specific to CH and needs moved, that is on the backlog somewhere

Reply:
I see, thanks @mathsusan